src/api/spec/lib/rubocop/cop/view_component/avoid_global_state_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+require './lib/rubocop/cop/view_component/avoid_global_state' # TODO: Why is Zeitwerk not auto-loading this?
```

Comment:
I'll address this by moving the custom cop under `src/api/app/lib/` instead of `src/api/lib/`.